pkg/shared/connection/api/defaultapi/default_client.go

```diff
@@ -73,16 +76,16 @@ func (a *defaultAPI) ServiceRegistryMgmt() registrymgmtclient.RegistriesApi {
 }
 
 // ServiceAccountMgmt return a new Service Account Management API client instance
-func (a *defaultAPI) ServiceAccountMgmt() kafkamgmtclient.SecurityApi {
+func (a *defaultAPI) ServiceAccountMgmt() svcacctmgmtclient.ServiceAccountsApi {
 	tc := a.CreateOAuthTransport(a.AccessToken)
-	client := kafkamgmt.NewAPIClient(&kafkamgmt.Config{
-		BaseURL:    a.ApiURL.String(),
+	client := svcacctmgmt.NewAPIClient(&svcacctmgmt.Config{
+		BaseURL:    build.ProductionAuthURL,
```

Comment:
I do not think this is right. I think we should connect it with the login flag we have and config. Build is compile time value

Reply:
@mikeedgar @andreaTP @riccardo-forina   for there will be some extra implications for UI, kas-instaler and srs-installer here.
Any other service that uses custom keycloak (not sso.redhat.com) will stop working when we merge that PR.
We were asked to migrate from kafka control plane service accounts  API to use Service Accounts provided by CIAM. This API has different url (not api.openshift.com). Service is hosted on gitlab - not sure if we can deploy this with kas-installer etc. Even if we deploy this keycloak url will still be used by all clients making it hard to integrate.

Reply:
+1 on making this configurable either via CLI flags or it being read dynamically from the control plane sso providers endpoint. 

> Any other service that uses custom keycloak (not sso.redhat.com) will stop working when we merge that PR.

I think @akoserwal is working on this. He may be able to chime in here with details.

Reply:
After initial discussion I think we good to assume the sso url as api root for this API

Reply:
> +1 on making this configurable either via CLI flags or it being read dynamically from the control plane sso providers endpoint.

+1 as well!

Reply:
> +1 on making this configurable either via CLI flags or it being read dynamically from the control plane sso providers endpoint.

Another +1. Having the base URL for a service configurable expands the ability to use services more easily outside of prod.